plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1881,7 +1882,10 @@ elementgui.armor.part_leg_right=leg R
 elementgui.armor.part_boot_left=boot L
 elementgui.armor.part_boot_right=boot R
 elementgui.armor.texture=Armor texture: 
-elementgui.armor.special_information=<html>Special information:<br><small>Use comma to separate entries
+elementgui.armor.helmet_special_information=<html>Special helmet information:<br><small>Use comma to separate entries
```

Comment:
I would actually reduce this to `<html>Special information:<br><small>Use comma to separate entries` that is used by all mod elements to reduce translations needed

Reply:
Done as well. Since we no longer use commas with @Defeatomizer's new system, I used existing key `elementgui.common.special_information` which only says `Special information`

Reply:
Good idea